### PR TITLE
Add Minecraft to supported sharing list, update SW / Gumball share text

### PR DESCRIPTION
**Pull request by bcjordan**
# Adds Minecraft to supported sharing list
- Add custom default sharing drawing
# Updates Star Wars and Gumball sharing text
## from

image
## to

image

### CODE REVIEW

File: apps/src/studio/skins.js [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 var skinsBase = require('../skins');
 var msg = require('./locale');
+var commonMsg = require('../locale');
```
Comment (breville) [inline apps/src/studio/skins.js:10]: we have two locales, a directory apart? :)

File: apps/src/studio/skins.js [hunk 2]
```diff
@@ -15,8 +16,15 @@ var HIDDEN_VALUE = constants.HIDDEN_VALUE;
...
 
+// Standard Twitter options matching defaults in FeedbackUtils.createSharingDiv
+// Use to avoid "story" reference in share text for a given skin.
+var plainTwitterOptions = {
+  text: commonMsg.defaultTwitterText() + " @codeorg",
+  hashtag: 'HourOfCode'
+};
 
 function loadGumball(skin, assetUrl) {
+  skin.twitterOptions = plainTwitterOptions;
```
Comment (breville) [inline apps/src/studio/skins.js:27]: I think we should do iceage as well?

**Comment by breville** [comment]
lgtm, though curious about iceage.  fyi @caleybrock 

thanks for doing this!

**Comment by bcjordan** [comment]
~~Added, TODO (me): locally test ice age before merge. Thanks for review!~~

Confirmed ice age:

image

**Comment by caleybrock** [comment]
:+1: thanks @bcjordan

**Pull request merged**